stripe-sms-receipt/functions/send-sms-receipt.js

```diff
@@ -0,0 +1,68 @@
+/**
+ *  Send Stripe SMS Receipt
+ *
+ *  This function shows you how to programmatically send a payment receipt URL to your customer via SMS,
+ *  using a Stripe webhook event as trigger.
+ *
+ *  Pre-requisites:
+ *  - Enable ACCOUNT_SID and AUTH_TOKEN in your functions configuration (https://www.twilio.com/console/functions/configure)
+ *  - Add STRIPE_SECRET_KEY to your environment variables (https://www.twilio.com/console/functions/configure)
+ *  - Add stripe to your NPM package dependencies (https://www.twilio.com/console/functions/configure)
+ */
+
+exports.handler = async function(context, event, callback) {
+  const response = new Twilio.Response();
+  response.appendHeader("Content-Type", "application/json");
+
+  // Uncomment to support CORS.
+  // response.appendHeader('Access-Control-Allow-Origin', '*');
+  // response.appendHeader('Access-Control-Allow-Methods', 'POST, OPTIONS');
+  // response.appendHeader('Access-Control-Allow-Headers', 'Content-Type');
+
+  // Only action charge.succeeded events
+  if (event.type === "charge.succeeded") {
+    // Retrieve the event object from Stripe to verify its validity.
+    // If you're running your own server environment you can verify
+    // the webhook signature instead:
+    // https://stripe.com/docs/webhooks/signatures#verify-official-libraries
+    try {
+      // To use stripe in your Twilio function you need to add it in the dependencies
+      // section of your functions config: https://www.twilio.com/console/functions/configure.
+      const stripe = require("stripe")(context.STRIPE_SECRET_KEY);
```

Comment:
If you move the require part of things (not the creation of the API client) outside of the handler function it will be cached while the Function is warm